### PR TITLE
Namespace error corrected in save_chat now running successfully

### PR DESCRIPTION
**Pull request by SinghHimadri**
## Issue that this pull request solves
#343 
Closes: # (issue number)
#343 
## Proposed changes

Brief description of what is fixed or changed
Namespace has been corrected and command can be executed successfully. Also, it was not mentioned in readme how to use this command. I have added that in Read me. This command runs successfully now. This setting file was created after entering this command and then to edit the credentials.Now the user can edit and run this command.
## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface
Screenshot from 2020-05-19 16-08-05

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ It is command line software through which you can play with your WhatsApp. It is
 
 ***message_timer*** is a message timer script. It sends messages to your WhatsApp contact from time to time. The number of messages and type of messages is decided by you. It's possible to send messages at random interval and the message type is chosen randomly.
 
-***save_chat*** is a script to save all the chat which we backup on our google drive.
+***save_chat*** is a script to save all the chat which we backup on our google drive.Edit the [auth] section in "settings.cfg".Google account login credentials (username and password) and Google services device id.Search Google Play for "device id" for plenty of apps that can reveal this information and enter your device id in deid in [auth] section.
```
Comment (rpotter12) [inline README.md:31]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5549409)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5549409&bid=18196136)

File: README.md (hunk 1 above)
Comment (rpotter12) [inline README.md:31]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5549409)

**Comment by SinghHimadri** [comment]
@rpotter12 Please have a look.

**Pull request closed without merging**